packages/reports/build/webpack.config.js

```diff
@@ -40,11 +40,13 @@ module.exports = function (env) {
                 'react-router-dom': 'react-router-dom',
                 'react-router': 'react-router',
                 mobx: 'mobx',
+                '@deriv/hooks': '@deriv/hooks',
```

Comment:
I added hooks to externals here because checks were still failing in the PR when trying to import useFeatureFlags from @deriv/hooks in reports package, even though the app was building without issues locally and in the test link. Open to suggestions.